com.dynamo.cr/com.dynamo.cr.bob.test/src/com/dynamo/bob/bundle/test/BundlerTest.java

```diff
@@ -589,6 +603,16 @@ public void testAbsoluteCustomResourcePath() throws IOException, ConfigurationEx
         return expectedFiles;
     }
 
+    private String getPathForBundleResource(Platform platform, String resource) {
```

Comment:
The APK format doesn't specifically prevent files in the root of the APK, but there is no way to access them, and when going from an AAB file to an APK it isn't even possible to put files in the root. (Well you could in a separate step but what is the point?)